### PR TITLE
Eliminate epm references

### PR DESCRIPTION
**Pull request by VoR0220**
* Eliminates all references to `epm`, replaces with `jobs`.

### CODE REVIEW

File: docs/specs/variable_specification.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ Variables come in the following types:
 
 The result of every job is set as a variable with the `JobName` as the key and the `JobResult` as the value. The `JobResult` for transaction jobs is the transaction hash. The `JobResult`  for contract deployments is the address of the contract. The `JobResult` for queries and calls is the return value from the blockchain or the query.
 
-The `JobResults` which are able to be retrieved from query functions will vary and depend largely on the fields which are returnable from mint-client's tooling.
+The `JobResults` which are able to be retrieved from query functions will vary and depend largely on the fields which are returnable from eris-db's tooling.
```
Comment (zramsay) [inline docs/specs/variable_specification.md]: `eris-client`

File: loaders/package.go [hunk 2]
```diff
@@ -28,19 +28,19 @@ func LoadPackage(fileName string) (*definitions.Package, error) {
...
-	}).Debug("Loading eris-pm file")
+	}).Debug("Loading eris jobs file")
 
 	epmJobs.AddConfigPath(path)
 	epmJobs.SetConfigName(bName)
 
 	// load file
 	if err := epmJobs.ReadInConfig(); err != nil {
-		return nil, fmt.Errorf("Sorry, the marmots were unable to load the eris-pm jobs file. Please check your path.\nERROR =>\t\t\t%v", err)
+		return nil, fmt.Errorf("Sorry, the marmots were unable to load the eris jobs file. Please check your path.\nERROR =>\t\t\t%v", err)
```
Comment (zramsay) [inline loaders/package.go]: fix these ugly error messages

File: loaders/package.go [hunk 3]
```diff
@@ -28,19 +28,19 @@ func LoadPackage(fileName string) (*definitions.Package, error) {
...
 	}
 
 	// marshall file
 	if err := epmJobs.Unmarshal(pkg); err != nil {
-		return nil, fmt.Errorf("Sorry, the marmots could not figure that eris-pm jobs file out.\nPlease check your epm.yaml is properly formatted.\n")
+		return nil, fmt.Errorf("Sorry, the marmots could not figure that eris jobs file out.\nPlease check your epm.yaml is properly formatted.\n")
```
Comment (zramsay) [inline loaders/package.go]: remove `\n`

Comment (VoR0220) [inline loaders/package.go]: Dammit you're right.

File: loaders/package.go (hunk 2 above)
Comment (VoR0220) [inline loaders/package.go]: I could have sworn those were already fixed....oh well...fixing.

File: docs/specs/variable_specification.md (hunk 1 above)
Comment (VoR0220) [inline docs/specs/variable_specification.md]: yea.

**Pull request merged**